static/app/data/experimentConfig.tsx

```diff
@@ -30,6 +30,12 @@ export const experimentList = [
     parameter: 'exposed',
     assignments: [0, 1],
   },
+  {
+    key: 'PrioritySortExperiment',
+    type: ExperimentType.ORGANIZATION,
```

Comment:
@ceorourke This should be type user, no?

Reply:
I can't get the feature to register if it's user, I mentioned it in the other PR